docs/articles/baseplatform/how-to-change-bootsplash.rst

```diff
@@ -0,0 +1,54 @@
+:orphan:
+
+Changing bootsplash
+===================
+
+PELUX uses psplash for graphical boot splash screen. The PELUX reference images come
```

Comment:
According to http://git.yoctoproject.org/cgit/cgit.cgi/psplash/tree/README, the naming `PSplash `shall be used iof `psplash`.

May occur at multiple locations in this doc.